fastlane/metadata/steam/french/description.txt

```diff
@@ -0,0 +1,59 @@
+Créez des chaînes d'approvisionnement complexes pour alimenter vos tourelles en munitions, produisez des matériaux à utiliser pour vos constructions et protégez vos structures des vagues d'ennemis. Jouez avec vos amis dans des parties multijoueurs cross-plateformes en co-op ou défiez-les lors de matchs PvP en équipe.
+
+[img]{STEAM_APP_IMAGE}/extras/ezgif-4-0e70c282f775.gif[/img]
+
+[h2]Gameplay[/h2]
+
+[list]
+[*] Créez des foreuses et des convoyeurs pour récolter et transporter les resources jusqu'à votre Noyau.
+[*] Utilisez les blocs de production avancés pour créer des ressources plus complexes.
+[*] Construisez des drones pour miner automatiquement les ressources, pour vous assistez dans la construction ou encore pour défendre votre base.
+[*] Gérez l'acheminent des liquides et combattez les incendies.
+[*] Boostez la production en approvisionnant vos blocs de défense et de production avec des liquides de refroidissement et des lubrifiants optionnels.
+[/list]
+
+[h2]Campagne[/h2]
+
+[list]
+[*] Avancez à travers les 12 zones rejouables de la campagne.
+[*] Récoltez and accumulez les resources.
+[*] Recherchez de nouveaux blocs et faites progresser les technologies à votre disposition.
+[*] Missions et objectifs variés.
+[*] Invitez vos amis pour terminer les missions ensemble.
+[*] 120+ blocs technologiques à maîtriser.
+[*] 19 types de drones, mechs et vaisseau.
+[*] 50+ succès à débloquer.
+[/list]
+
+[h2]Modes de jeu[/h2]
+
+[*] [b]Survie[/b]: Construisez des tourelles pour défendre votre base des ennemis dans un style de jeu inspiré des tower-defense. Survivez aussi longtemps que possible, avant de faire décoller votre noyau. Utiliser les ressources récoltées pour débloquer de nouvelles technologies. Préparez votre base pour les attaques de puissants boss aériens.
+[*] [b]Attaque[/b]: Construisez des usines et produisez des unités pour détruire les noyaux ennemis, tout en défendant votre base contre les vagues d'attaque. Créez une grande variété d'unités offensives et de support pour vous aider dans vos objectifs.
+[*] [b]PvP[/b]: Affrontez d'autres joueurs. Constituez jusqu'à 4 équipes différentes et tentez ed détruire les noyaux adverses. Créez des unités d'assaut ou attaquez directement les autres bases avec vos mechs.
```

Comment:
Typo here : "tentez de détruire" is correct

Reply:
Fixed
Thanks